src/mixins/numeral-formatable.js

```diff
@@ -0,0 +1,267 @@
+import {
+  allowedMasks
+} from '../util/mask'
+
+export default {
+  data: () => ({
+    options: {},
+    signSymbol: '',
+    preDefined: {
+      'numeral': {
+        positiveOnly: false, // Positive value only
+        positiveSign: false, // Show positive sign '+'
+        blockSeparator: ' ', // {String|Array} - Separate between groups of digits
+        blockSize: 3, // {Number|Array} - Number of digits per group
+        decimal: '.', // Decimal mark
+        prefix: '', // Prefix shown in input field
+        precision: 2, // Floating point precision
+        suffix: '', // Suffix shown in input field
+        formatter: 'numeral'
+      }
+    }
+  }),
+
+  computed: {
+    precision () {
+      return this.absVal(this.options.precision)
+    },
+    positiveSign () {
+      return this.options.positiveSign ? '+' : ''
+    },
+    decimal () {
+      return this.oneChar(this.options.decimal)
+    },
+    blockSeparator () { // Default separator = ' '
+      const separator = this.options.blockSeparator
+
+      if (!Array.isArray(separator)) return [this.oneChar(separator, '')]
+
+      switch (separator.length) {
+        case 0: return [' ']
+        case 1: return this.oneChar(separator, '')
```

Comment:
isn't it just `separator[0]`?
Which leads to the 
```js
return separator.length ? separator : [' ']
```
solution

Reply:
Well, I think it's probably better not to validate any of the options at all because it's the dev's responsibility to define it correctly if he wants his codes to run well.